### DISCUSSION TITLE
I don't know why my application always be rejected,I have applied 29 times!!!

### DISCUSSION THREAD

**Discussion by WangSiwei2022080233** [community/community#53385] [GitHub Education] [answered]
see picture. my profile is 40 letters enough,I don' want to say anymore.[](url)
![Uploading 屏幕截图 2023-04-21 153954.png…]()

**Comment by samus-aran** [comment, marked as answer]
Hi @WangSiwei2022080233  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by WangSiwei2022080233** [reply]
I have been approved! thank you very much.the problem was my username.
